phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 6
+    paths:
+        - bin/
+        - config/
+        - public/
+        - src/
+        - tests/
```

Comment:
You're not allowed to touch to this file

Reply:
I just download the ux/component extension I don't touch anything here